docs/intro-to-lora.md

```diff
@@ -45,7 +45,9 @@ For development purposes one gateway will be enough, but in a production deploym
 
 We'll also need to build devices. If you want to use ARM mbed (and you should) there are two options:
 
-* Get the [Multitech mDot](https://developer.mbed.org/platforms/MTS-mDot-F411/) and the [EVK board](http://www.digikey.com/product-detail/en/multi-tech-systems-inc/MTUDK2-ST-MDOT/591-1278-ND/5247463).
+* Get the [Multitech mDot](https://developer.mbed.org/platforms/MTS-mDot-F411/) and the [UDK2 board](http://www.digikey.com/product-detail/en/multi-tech-systems-inc/MTUDK2-ST-MDOT/591-1278-ND/5247463).
+	* As an alternative, you can use the [mDot EVB](https://developer.mbed.org/platforms/mdotevb/) which is the mDot reference design. 
```

Comment:
As an alternative, you can use the [mDot EVB](https://developer.mbed.org/platforms/mdotevb/), which is the mDot reference design.